cpp/src/Slice/Parser.cpp

```diff
@@ -648,48 +648,44 @@ namespace
         if (stripMarkup)
         {
             // Strip HTML markup.
```

Comment:
Switching to a while loop lets us avoid a redundant check for `pos != string::npos`.

Reply:
The code assumes anything between `<` and `>` is an html tag.

What about something like:

`a < b or x > y` , won't this strip comments like to `a  y`?

It is not a new problem, but would be nice to fix now that we are looking at this code.

We can fix in a separate PR.